README.md

```diff
@@ -74,6 +74,7 @@ A curated list of amazingly awesome open source resources for broadcasters inspi
 
 ## Multimedia content processing
 
+* [codem-isoboxer] (https://github.com/madebyhiro/codem-isoboxer) A small browser-based MPEG-4 (ISOBMFF) parser.
```

Comment:
1. Could you please capitalize the first letter of the project.
2. Not sure about the category, since it is a browser package, shouldn't it go to the Media Player section?